src/renderer/components/modals/create-note-modal/CreateNoteModal.vue

```diff
@@ -10,12 +8,14 @@ const noteNameCharacters = 'abcdef0123456789';
 
 const generateNoteName = () => {
   let text = '';
-  for (let i = 0; i < 32; i += 1)
+  for (let i = 0; i < 32; i += 1) {
     text += noteNameCharacters.charAt(Math.floor(Math.random() * 16));
+  }
   return text;
 };
 
 export default {
+  template: require('./CreateNoteModal.html'),
```

Comment:
is it the new way to declare template?

Reply:
This was done to fix an error with the vue-loader. I can investigate an alternative fix, but this works.